dcos/dcos-ui
# DCOS-46413: Server side error message for invalid input in package creation does not go away after updating the field

Hide cosmos errors when editing the form.

Closes https://jira.mesosphere.com/browse/DCOS-46413

## Testing
1. Go to Catalog tab.
2. Select a package and enter some error in the configuration that Cosmos would catch, for example in chronos, go to "Chronos" tab and enter "/Chronos" as ID (invalid because of capital letter).
3. Go to "Review & Run" and try to submit.
4. An error should appear. As you go back, the error should disappear. It shouldn't appear even when going back to the review screen before clicking the submit button.

## Trade-offs
My initial wish was to use `onCosmosPackagesStoreServiceDescriptionError` from `EditFrameworkConfiguration.js` to clear errors, but I couldn't pass it as props for some reason.

## Dependencies
None.

## Screenshots

### Before
![Peek 2019-03-21 15-21](https://user-images.githubusercontent.com/40791275/54755800-6f6a1180-4bef-11e9-9d8c-064416263c86.gif)

### After
![Peek 2019-03-21 15-19](https://user-images.githubusercontent.com/40791275/54755790-6aa55d80-4bef-11e9-83aa-cba54cc58c8e.gif)
